### ISSUE TITLE
New 64-bit compatibility rules

### ISSUE THREAD

**Issue by rpurushotma** [freshplanet/ANE-Alert#15]
With the new AIR SDK updated to meet Apple's new 64 bit requirements, I started getting the following error:

_Error occurred while packaging the application:
Error: META-INF/ANE/iPhone-ARM/libAirAlert.a, are required to have universal iOS libraries. Please contact the ANE developer(s) to get the same._

Any chance of recompiling it with the AIR 16 Beta SDK?

**Comment by therabbitwindfall** [comment]
I will check today. Compiled but not tested yet. Will post a link here

**Comment by therabbitwindfall** [comment]
Here is repack that supports iOS x64 ARM
https://www.dropbox.com/s/6ykc9wg9zdvxcnj/AirAlert.ane

**Comment by thomascallahan** [comment]
Is there anything else we have to do to get this working? I put it in my project, linked the ANE, and dropped in the import statement like this:

	import com.freshplanet.ane.AirAlert;

But I get "access of possibly undefined propert AirAlert" errors, and I'm not getting autocomplete on the import statement so I think it's not actually recognizing the ANE correctly or something. FlashBuilder 4.7, AIR SDK 16 latest beta, Windows 8.1 x64.

**Comment by therabbitwindfall** [comment]
Actually I don't know because my repacked ANE that I post above works correctly with Flash CC in Windows 7 x64. Original ANE work for you?

![2015-01-13_002701](https://cloud.githubusercontent.com/assets/1976931/5711367/ed6b1fa6-9aba-11e4-83be-d36e648ba410.jpg)

**Comment by thomascallahan** [comment]
Hmm. No it doesn't seem to. Is it "com.freshplanet.ane*.AirAlert*" or 
"com.freshplanet.ane*.AirAlert.AirAlert*"? Neither works for me, but if 
I have two "AirAlert"s in there I get "definition not found" errors 
where with just one, I don't.

I must be having other issues -- this was an old project I hadn't really 
planned on updating but it has some layout issues on iPhone 6's so 
installed FlashBuilder just for this. I think my installation is bad.

Plus I'm realizing I need more than just 2 buttons -- I need three 
buttons in one place, an input field in at least one place, and a 
generic spinner in a couple. Might have to resort to one from distriqt 
or Milkman but I can't find anything saying theirs are Universal.

-Tom

**Comment by therabbitwindfall** [comment]
Original and recompiled work for me as on my screenshot above :)
try to do this:
1) open ANE in any ZIP archiver
2) extract library.swf and catalog.xml
3) create new AirAneSWC.zip file and put inside files from step 2 and rename it to AirAneSWC.swc
4) Try to link new SWC file for definition and put also extension. In this case definition could work for you and extension will work the same.

But I don't know why ANE's don't works simple for you :) This extension don't show any issues on iPhone6

**Comment by ionutmihaila** [comment]
Yes, it's working, thank you! But only if you remove the old AirFacebook.ane first.
